arboretum/ibm_cloud/README.md

```diff
@@ -62,6 +62,49 @@ to include the fetchers and checks from this library in your downstream project.
    from arboretum.ibm_cloud.fetchers.fetch_cluster_list import ClusterListFetcher
    ```
 
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-ibm-cloud-cluster-resource]
+* Purpose: Write the resources of IBM Cloud Kubernetes clusters to the evidence locker.
+* Behavior: Retrieve the resources of IBM Cloud Kubernetes clusters listed by [cluster list fetcher][fetch-cluster-list]. For IBM Cloud Kubernetes Service (IKS) clusters, the fetcher [downloads cluster config via IBM Cloud API][ibm-cloud-download-config] and extract the access token from the config. For IBM Cloud Red Hat Kubernetes Service (ROKS) clusters, the fetcher retrieves the access token using [the OAUTH server][ibm-cloud-roks-oauth] of the cluster. TTL is set to 1 day.
```

Comment:
Is it relevant to call out how the access token is retrieved? This behavior doesn't describe anything having to do with the fetcher behavior but more about how access seems to be acquired to perform the fetch.

Reply:
I removed the description of internal behavior and move it to code comments of [IKS cluster function](https://github.com/ComplianceAsCode/auditree-arboretum/pull/54/files#diff-bcd498e6b9c862fab4f20772411039245ac766ae4cab4186d5c9285b8835064aR66-R67) and [ROKS cluster function](https://github.com/ComplianceAsCode/auditree-arboretum/pull/54/files#diff-bcd498e6b9c862fab4f20772411039245ac766ae4cab4186d5c9285b8835064aR87-R88).